barricade/src/main/java/com/mutualmobile/barricade/activity/BarricadeActivity.java

```diff
@@ -65,15 +65,19 @@ private void initUi() {
   }
 
   private void setEndpointsView() {
-    actionBar.setTitle(R.string.title_activity);
+    if (actionBar != null) {
```

Comment:
create a method called setActionBarTitle() which internally checks if actionBar is null or not and then sets the title on the actionBar. Use that method here and in line 78